lib/views/general/_footer.html.erb

```diff
@@ -30,3 +30,4 @@
         <div style="clear:both"></div>
     </div>
 </div>
+<div class="after-footer">&nbsp;</div>
```

Comment:
Not sure why this has been added? Can this partial be deleted given they're on the responsive theme now?